acceptance/reproducibility_test.go

```diff
@@ -96,8 +96,14 @@ func testReproducibility(t *testing.T, when spec.G, it spec.S) {
 
 	it.After(func() {
 		// clean up any local images
-		h.DockerRmi(dockerClient, imageName1)
-		h.DockerRmi(dockerClient, imageName2)
+		err := h.DockerRmi(dockerClient, imageName1)
+		if err != nil {
+			fmt.Print("error ", err)
+		}
+		err := h.DockerRmi(dockerClient, imageName2)
+		if err != nil {
+			fmt.Print("error ", err)
+		}
```

Comment:
```suggestion
		h.AssertNil(t, h.DockerRmi(dockerClient, imageName1))
		h.AssertNil(t, h.DockerRmi(dockerClient, imageName2))
```

Reply:
@jromero I recall you had some reservations about adding error checks in test cleanup steps. Do those apply here?

Reply:
We opted to keep the error checks (as they are valueable) but continue execution. We did this via a new helper method. `h.AssertNilE`.  See https://github.com/buildpacks/pack/pull/1162/files

Reply:
I like what @jromero wrote that they did in pack.
I think it's better to print an error but continue the execution in this case by creating a similar helper like `h.AssertNilE`.
The difference between `AssertNil` and `AssertNilE` is:
* `AssertNil` checks for nil value, if not nil it fails the test and stops execution immediately.
* `AssertNilE` checks for nil value, if not nil it sets test as failed without stopping execution.

@natalieparellano, do you agree?

Reply:
I like this idea. It might be a little unfair to make that a requirement of this PR, since we're already doing error checks in some of our cleanups. But maybe we could make an issue for it.

Reply:
Made an issue for adding `AssertNilE`: https://github.com/buildpacks/imgutil/issues/121